corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -296,6 +300,40 @@ hqDefine('app_manager/js/app_manager.js', function () {
                 $(related).data(name, value);
             });
         }
+        function resetIndexes($sortable) {
```

Comment:
Feel like I'm missing something big, but isn't this unnecessary now that the view module / view form / form builder links are all based on unique id, not on index?